src/main/java/de/muenchen/isi/api/dto/calculation/WohneinheitenBedarfDto.java

```diff
@@ -0,0 +1,18 @@
+package de.muenchen.isi.api.dto.calculation;
+
+import java.math.BigDecimal;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class WohneinheitenBedarfDto {
```

Comment:
Sind die Klassen ` WohneinheitenBedarf` und `PlanungsursaechlicherBedarf` korrekt benannt?

Die beiden Klassen geben die aus den Bauraten ermittelten Wohneinheiten zurück und nicht den Bedarf an Wohneinheiten.

Ein Bedarf an Wohneinheiten würde sich z.B. aus einer Anzahl an Einwohner ergeben.

Reply:
Witzigerweise wird in der ISI-88 der langfristige planungsursächliche Bedarfe an Kinderkrippen bzw- -garten ermittelt. 

Hier habe ich die Klassen `LangfristigerPlanungsursaechlicherBedarf` welche eine Liste an `PlanungsursaechlicherBedarf` beinhaltet